docs/tutorials/quick_start.md

```diff
@@ -13,31 +13,27 @@ paths:
 
 In this article, you will find the very basics of using the Apify platform. It shows you the principles of running any actor in Apify Store and the basics of making them using the in-app code editor.
 
-Once you get to grips with this, [go through our Apify scrapers tutorials]({{@link tutorials/apify_scrapers.md}}) to learn about how our main tools work. If you're comfortable getting your hands into some JavaScript, [check out the Apify SDK](https://sdk.apify.com), which has plenty of [tutorials to get you going](https://sdk.apify.com/docs/guides/quick-start).
+When you're done here, [see our Apify scrapers tutorials]({{@link tutorials/apify_scrapers.md}}) to learn about our main tools. If you're comfortable using JavaScript, [check out the Apify SDK](https://sdk.apify.com), which has plenty of [tutorials to get you going](https://sdk.apify.com/docs/guides/quick-start).
 
 **You will need an Apify account to complete this tutorial. If you don't have one, [complete the sign-up process](https://console.apify.com/sign-up) first.**
 
 ## Run an actor
 
-**1.** Navigate to [apify.com/store](https://apify.com/store) and pick any actor you like the look of. We'll pick the [Google Search Results Scraper](https://apify.com/apify/google-search-scraper).
+**1.** Navigate to [Apify Store](hhttps://console.apify.com/actors#/store/) and pick any actor you like the look of. We'll pick the [Google Search Results Scraper](https://apify.com/apify/google-search-scraper).
```

Comment:
This is weird, how one link puts you to console, and another to web, we should have both leading to the same place.